modules/ROOT/pages/install-acm.adoc

```diff
@@ -4,9 +4,9 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-NOTE: Installing, updating or configuring API Community Manager must be done by a user account with the *System Administrator* profile in the ACM Salesforce organization.
+NOTE: Installing, updating or configuring API Community Manager (ACM) must be done by a user account with the *System Administrator* profile in the ACM Salesforce organization.
 
-To perform a new install of the API Community Manager package in your ACM Salesforce Organization, follow the steps below. To update an existing API Community Manager installation, refer to xref:update-acm.adoc[Update ACM].
+To perform a new install of the API Community Manager package in your ACM Salesforce Organization, follow the steps below. To update an existing ACM installation, refer to xref:update-acm.adoc[Update ACM].
```

Comment:
install operation or installation, not install

Reply:
Note that organization is not capitalized in previous text but is here; GLOBALLY standardize. I suspect this should be lowercase.

Reply:
GLOBALLY reword to not refer to anything "above" or "below." Use following, next, previous, earlier, and so on.